Firebase.agent.lib.nut

```diff
@@ -570,7 +614,7 @@ class Firebase {
     }
 
     // return a Promise if the Promise library is included
-    function _returnPromise (request){
+    function _returnPromise(request) {
         if (_promiseIncluded) {
             return Promise(function (resolve,reject){
                     request.sendasync(function(res){
```

Comment:
Do we need to have the same checks for null and zero lengths as in the `_invokeCallback` function?